speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/racescreens/raceresultsmenus.cpp

```diff
@@ -224,7 +224,7 @@ rmPracticeResults(void *prevHdle, tRmInfo *info, int start)
     GfuiMenuCreateButtonControl(rmScrHdle, hmenu, "ContinueButton", prevHdle, GfuiScreenReplace);
 
     // SIMULATED DRIVING ASSISTANCE: Add "Quit" button
-    GfuiMenuCreateButtonControl(rmScrHdle, hmenu, "QuitButton", EndExperimentInit(), GfuiScreenActivate);
+    GfuiMenuCreateButtonControl(rmScrHdle, hmenu, "QuitButton", EndExperimentInit(RACE_EXIT), GfuiScreenActivate);
```

Comment:
I'm pretty sure this code never gets called anymore since we dont run in practice mode.

Reply:
see comment above